src/vs/platform/actions/browser/actionWidgetDropdownActionViewItem.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { $, append } from '../../../base/browser/dom.js';
+import { BaseActionViewItem } from '../../../base/browser/ui/actionbar/actionViewItems.js';
+import { ILabelRenderer } from '../../../base/browser/ui/dropdown/dropdown.js';
+import { getBaseLayerHoverDelegate } from '../../../base/browser/ui/hover/hoverDelegate2.js';
+import { getDefaultHoverDelegate } from '../../../base/browser/ui/hover/hoverDelegateFactory.js';
+import { IAction } from '../../../base/common/actions.js';
+import { IDisposable } from '../../../base/common/lifecycle.js';
+import { IActionWidgetService } from '../../actionWidget/browser/actionWidget.js';
+import { ActionWidgetDropdown, IActionWidgetDropdownOptions } from '../../actionWidget/browser/actionWidgetDropdown.js';
+
+/**
+ * Action view item for the custom action widget dropdown widget.
+ * Very closely based off of `DropdownMenuActionViewItem`, would be good to have some code re-use in the future
+ */
+export class ActionWidgetDropdownActionViewItem extends BaseActionViewItem {
```

Comment:
I'm not sure now if I like this name more or @Yoyokrazy 's `NotebookOutputEditorInput` ;)